pkg/build/builder/daemonless.go

```diff
@@ -305,32 +309,101 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 
 func generateTransientMounts() []string {
 	mounts := []string{}
-	mounts = appendRHSMMount(mounts)
+	mounts = appendRHSMMount(defaultMountStart, mounts)
+	mounts = appendETCPKIMount(defaultMountStart, mounts)
+	mounts = appendRHRepoMount(defaultMountStart, mounts)
 	mounts = appendCATrustMount(mounts)
 	return mounts
 }
 
-func appendRHSMMount(mounts []string) []string {
-	st, err := os.Stat("/run/secrets/rhsm")
+func appendRHRepoMount(pathStart string, mounts []string) []string {
+	path := filepath.Join(pathStart, "redhat.repo")
```

Comment:
`redhat.repo` is used several places here, would it be worth using a constant for this as well?

Reply:
I'll do it for all 3 files / dirs ... part of next push